backend/main.py

```
@@ -361,6 +363,234 @@ def get_all_products(db: Session = Depends(get_db)):
+        status="Pending"
+    )
+    db.add(db_bill)
+    db.commit()
+    db.refresh(db_order)
+    return db_order
+
+
+
+
+@app.get("/orders/me")
+def get_my_orders(
```

Ruff (0.8.2)

Missing return type annotation for public function `get_my_orders`

(ANN201)